manim/animation/creation.py

```diff
@@ -251,6 +290,9 @@ class AddTextLetterByLetter(ShowIncreasingSubsets):
     time_per_char : :class:`float`
         Frequency of appearance of the letters.
 
+    .. tip::
```

Comment:
Missing newline after the directive.

```suggestion
    .. tip::

```